modules/web-mercator/src/web-mercator-viewport.js

```diff
@@ -30,7 +32,8 @@ export default class WebMercatorViewport {
       zoom = 0,
       pitch = 0,
       bearing = 0,
-      altitude = 1.5,
+      altitude = DEFAULT_ALTITUDE,
+      fovy = altitudeToFovy(DEFAULT_ALTITUDE),
```

Comment:
This is a breaking change. If only altitude is specified then fov should be calculated from altitude, not the default value. If only fov is specified then altitude should be set to focalDistance.

Reply:
I've added `fovyToAltitude()` and adjusted the defaults based on your logic